client/app/components/account_config_main.coffee

```diff
@@ -0,0 +1,471 @@
+{
+    div, p, h3, h4, form, label, input, button, ul, li, a, span, i,
+    fieldset, legend
+} = React.DOM
+classer = React.addons.classSet
+
+AccountActionCreator = require '../actions/account_action_creator'
+AccountInput = require './account_config_input'
+
+RouterMixin = require '../mixins/router_mixin'
+LayoutActionCreator = require '../actions/layout_action_creator'
+{Form, FieldSet, FormButtons, FormDropdown} = require './basic_components'
+
+
+state = null
```

Comment:
It's a copy paste from the previous code, I was afraid to break something by removing it.